docs/patterns/deferredcallbacks.rst

```diff
@@ -56,7 +56,7 @@ this the following function needs to be registered as
 A Practical Example
 -------------------
 
-Now we can easily at any point in time register a function to be called at
+Now, at any point in time, we can easily register a function to be called at
```

Comment:
"At any time during the request, we can register a function to be called at the end of the request."